src/HTMLGenerator/converter.py

```diff
@@ -60,3 +64,8 @@ def convert(top_level_output_directory, top_level_input_directory, json_with_fil
     os.remove(current_configuration_absolute_path)
 
     return
+
+def convertFromCli(jsonFilePath) :
+    with open(jsonFilePath) as jsonFile:
+            json_with_filepaths = json.load(jsonFile)
+            convert(Path(""),Path(""),json_with_filepaths,Path.cwd())
```

Comment:
Why not set these empty path values as default on the parameters of `convert()`, like you suggested the other day?

Reply:
I did not change it because the function call in sequencer.py also had to be changed. I have changed it in the new commit.

Reply:
So in python the default formal parameters need to be after the non assigned parameters. Thus, json_with_filenames will become the first parameter and we will have to change the actual parameters for the function in sequencer.py

Reply:
Cool, we can leave that unchanged, then.